sidebar.js

```diff
@@ -133,6 +133,20 @@ const sidebars = {
       items: [{ type: "autogenerated", dirName: "scripting/trust_scripts" }],
     },
   ],
+  jargonSidebar: [
```

Comment:
need to find the right spot for this guy.

It will be good to include #252 as part of this PR. So let's include that work in this and figure out where to put this file.

Reply:
Updated the PR title as well as the description to include the work for #252